pkg/asset/manifests/ovirt/cloudproviderconfig.go

```diff
@@ -0,0 +1,26 @@
+package ovirt
+
+// CloudProviderConfig represents ovirt's cloud provider config map
+type CloudProviderConfig struct {
+	URL               string `json:"ovirt_url,omitempty"`
+	Username          string `json:"ovirt_username,omitempty"`
```

Comment:
cannot accept a cloud provider config that contains secret information like username/password.

Reply:
@abhinavdahiya, @rgolangh & I met and it seems the entire cloud provider config can be removed.